sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/BUILD

```diff
@@ -113,25 +115,27 @@ _CP_PROVISIONING_CMD_ARGS = """
 
 _CP_PROVISIONING_HARNESS = "//sw/host/provisioning/cp"
 
+_CP_PROVISIONING_CW_PARAMS = cw310_params(
+    binaries = {":sram_cp_provision": "sram_cp_provision"},
+    changes_otp = True,
+    needs_jtag = True,
+    otp = "//hw/ip/otp_ctrl/data/earlgrey_a0_skus/sival_bringup:otp_img_test_unlocked0_manuf_empty",
+    tags = [
+        "manuf",
+    ],
+    test_cmd = _CP_PROVISIONING_CMD_ARGS,
+    test_harness = _CP_PROVISIONING_HARNESS,
+)
+
 opentitan_test(
     name = "cp_provision",
+    cw310 = _CP_PROVISIONING_CW_PARAMS,
```

Comment:
ever since #23276 was merged, I _think_ this should just be `fpga = fpga_params` and that will use the same params for both cw310 and cw340. I.e., there is no need to specify both anytmore.

Reply:
@jwnrt can probably confirm here

Reply:
For clarity, you can define the parameters once under the name `fpga` and they will be used for both kinds of FPGA execution environments by default:

```python
opentitan_test(
    name = "cp_provision",
    fpga = fpga_params(
        binaries = {":sram_cp_provision": "sram_cp_provision"},
        ...
    ),
    exec_env = {
        "//hw/top_earlgrey:fpga_cw310_sival": None,
        "//hw/top_earlgrey:fpga_cw340_sival": None,
    },
    ...
)
```

The `None` here tells the execution environments to use the default parameter with the name `fpga`.

Reply:
`fpga = fpga_params` was already in #23276 which seems that this commit is not necessary. Is that correct?

Reply:
I think you still need to add `"//hw/top_earlgrey:fpga_cw340_sival": None,` and `"//hw/top_earlgrey:fpga_cw340_rom_with_fake_keys": None,` to your `exec_envs`, the changes to parameters should no longer be necessary

Reply:
Oops sorry I didn't refresh the page before leaving the comment. Will update the exec_env later by this commit. 
Thank you for the help.